jekyll/_cci2_ja/docker-hub-pull-through-mirror.adoc

```diff
@@ -1,136 +1,155 @@
+NOTE: *このサーバーはホストネームでアクセスできるようにする必要があります*。 つまり、ホスト名 (`your-mirror.example.com` とします) がこのサーバーの IP アドレスに正しく解決されるように、DNS を設定する必要があります。 In this guide we assume that the URL for the registry we are setting up is `\https://your-mirror.example.com` (be aware that the URL scheme is `http**_s_**`, not `http`). この URL は後で Nomad クライアントと VM サービスを設定する際に必要になります。
 
-. *Obtain a TLS certificate for `your-mirror.example.com`* and *put the certificate and the private key into `/root/tls`* of the server.
+. *`your-mirror.example.com` の TLS 証明書を取得し*、*TLS 証明書とプライベートキーをサーバーの `/root/tls`* に置きます。
 +
-NOTE: This guide assumes you are obtaining a certificate issued by a well-known CA, not a self-signed certificate. Use of self-signed certificates has NOT been tested.
+NOTE: このガイドでは、自己署名証明書ではなく、既知の CA が発行した証明書を取得することを想定しています。 自己署名証明書の使用はまだテストされていません。
 
-. 以下のコマンドを実行して、*キャッシュ サーバーを起動* します。
+. 以下のコマンドを実行して、*レジストリを起動*します。
 +
 [NOTE]
 ====
-`DOCKER_HUB_USERNAME` は Docker Hub アカウントのユーザー名に、`DOCKER_HUB_ACCESS_TOKEN` は https://hub.docker.com/settings/security で取得したアクセス トークンに置き換えてください。
+`DOCKER_HUB_USERNAME` は Docker Hub アカウントのユーザー名に、`DOCKER_HUB_ACCESS_TOKEN` は https://hub.docker.com/settings/security で取得したアクセストークンにそれぞれ置き換えてください。
 
-Replace `fullchain.pem` and `privkey.pem` with the filenames of your certificate and private key, respectively. *The certificate specified there must be correctly chained enough for tracing the certification path from the leaf to the root*.
+`fullchain.pem` は証明書のファイル名に、`privkey.pem` はプライベートキーのファイル名にそれぞれ置き換えてください。 *ここで指定する証明書は、リーフからルートへの証明書パスの追跡が可能な正確にチェーンされた証明書である必要があります*。
 ====
 +
 [source,bash]
 ----
-sudo bash -c 'cat <<< $(jq ".¥"registry-mirrors¥" = [¥"http://192.0.2.1¥"]" /etc/docker/daemon.json) > /etc/docker/daemon.json'
+sudo docker run \
+  -d \
+  -p 443:5000 \
+  --restart=always \
+  --name=through-cache-secure \
+  -v /root/tls:/data/tls \
+  -e REGISTRY_PROXY_REMOTEURL="https://registry-1.docker.io" \
+  -e REGISTRY_PROXY_USERNAME=DOCKER_HUB_USERNAME \
+  -e REGISTRY_PROXY_PASSWORD=DOCKER_HUB_ACCESS_TOKEN \
+  -e REGISTRY_HTTP_TLS_CERTIFICATE=/data/tls/fullchain.pem \
+  -e REGISTRY_HTTP_TLS_KEY=/data/tls/privkey.pem \
+  registry
 ----
 
-. Finally, *make sure that the TCP port 443 (i.e. HTTPS) is open and accessible*. For better security, we recommend you to open the port only to Nomad clients and VMs for `machine` executors and remote docker engines.
+. Finally, *make sure that the TCP port 443 (i.e. HTTPS) is open and accessible*. セキュリティーを強化するために、Nomad クライアントと `machine` とリモート Docker Engine の VM にのみポートを開くことをお勧めします。
```

Comment:
There is no translated part.

Reply:
Thank you! This part will be updated next week.